File: 6/Problem3.c
#include<stdio.h>

void DisplayFactorial(int iNo)
{
    int iCnt = 0;
    int iFact = 1;
    if(iNo < 0)
    {
        iNo = -iNo;
    }
    for(iCnt = 1; iCnt <= iNo; iCnt++)
    {
        iFact = iFact * iCnt;
    }
    printf("Factorial is : %d\n",iFact);
    
}
int main()
{
    int iValue = 0;
     printf("Enter the number : ");
     scanf("%d",&iValue);

     DisplayFactorial(iValue);

     return 0;
}

File: 18/problem3.java
import java.util.*;

class DigitX
{
    public void OddNum(int iNo)
    {
        int iCnt = 0;
        for(iCnt = 1 ; iCnt <= iNo; iCnt++)
        {
            if(iCnt % 2 != 0)
            {
                System.out.println(+iCnt);
            }
        }
        
    }
}
class problem3
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        DigitX dobj = new DigitX();

        int iValue = 0;

        System.out.println("Enter the number : ");
        iValue = sobj.nextInt();

        dobj.OddNum(iValue);

    }
}